crates/pallet-domains/src/lib.rs

```diff
@@ -1931,13 +1937,23 @@ mod pallet {
                 let consensus_transaction_byte_fee =
                     sp_domains::DOMAIN_STORAGE_FEE_MULTIPLIER * transaction_byte_fee;
 
-                let invalid_inherent_extrinsic_data = InvalidInherentExtrinsicData {
-                    extrinsics_shuffling_seed,
-                    timestamp,
-                    consensus_transaction_byte_fee,
-                };
+                for domain_id in SuccessfulBundles::<T>::iter_keys() {
```

Comment:
Is this the right logic, or should there be another list of domains I’m iterating through?
What happens if there aren’t any domains at all?

Reply:
I added commit [1be717e](https://github.com/autonomys/subspace/pull/3327/commits/1be717e3d39296e8db4db67da0414e8b3e71ab74) to also iterate through the runtime upgrades.